backend/config/views.py

```diff
@@ -1,15 +1,17 @@
 from graphene_django.views import GraphQLView
 from sentry_sdk.api import start_transaction
+from graphql import get_operation_ast, parse
 
 
 class CustomGraphQLView(GraphQLView):
     def execute_graphql_request(self, request, data, query, variables, operation_name, show_graphiql=False):
         """
         Enables performance tracing for GraphQL calls
         """
-        backend = self.get_backend(request)
-        if backend and query:
-            operation_type = backend.document_from_string(self.schema, query).get_operation_type(operation_name)
+        document = parse(query)
+        operation_ast = get_operation_ast(document, operation_name)
+        if operation_ast and query:
+            operation_type = operation_ast.operation
```

Comment:
`get_backend` is deprecated, might be simpler ways of obtaining the operation type